packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -490,196 +536,151 @@ export function getValidatorApi({
       });
       logger.debug("Produced common block body", loggerContext);
 
+      logger.info("Block production race (builder vs execution) starting", {
```

Comment:
Do we always wanna log this or only when we do the actual race because both builder and engine are enabled? I guess log context has all the information regarding builder / engine enabled

Reply:
changed to verbose log